src/frontend/components/DashboardJoinDiscussion/index.spec.tsx

```diff
@@ -40,19 +44,26 @@ describe('<DashboardJoinDiscussion />', () => {
 
     // reject the request
     fireEvent.click(rejectJohnDoe);
-    act(() => {
-      // remove participant 1 from the waiting list
-      useJoinParticipant.getState().removeParticipantAskingToJoin(participant1);
-    });
+    rerender(
+      wrapInIntlProvider(
+        <DashboardJoinDiscussion
+          video={{ ...video, participants_asking_to_join: [] }}
+        />,
+      ),
+    );
```

Comment:
clicking is useless, I agree.
But it was already in place when i edited this test. I can remove them anyway.
The rerender reflects what will happen in reality, as the video will mutate after an update through websocket.

Reply:
Inded, my point is you have two ways of testing this:
- omit the click and rerender the component
- mock converse method and mock return values of these method and do not rerender since the update will happen anyway if you wrap your test to be able to have a integration test for this component

Reply:
Here is the way it works :
when we click on rejectJohnDoe, for example, an API call is made for updating the current participant list.
Then the server sends the updated video through websocket, and an parent frontend component will update the current video object.
So a rerender will happen.
I'm removing the clicks from the test :)